### DISCUSSION TITLE
release notes?

### DISCUSSION THREAD

**Discussion by niXman** [jedisct1/libsodium#1064] [Q&A] [answered]
Hello,

where can I see the release notes/changelogs for releases?
I don't see this on release tags.

thanks!

**Comment by jedisct1** [comment, marked as answer]
In the `ChangeLog` file.

**Reply by niXman** [reply]
that is, in order to decide whether I need to update the library or not I need to download it first?
not entirely logical, don't you think so? ;)

**Reply by ektrah** [reply]
https://github.com/jedisct1/libsodium/blob/master/ChangeLog
